tests/lib/testers/rule-tester.js

```diff
@@ -126,7 +126,7 @@ describe("RuleTester", () => {
                     { code: "var foo = bar;", errors: [{ message: "Bad error message." }] }
                 ]
             });
-        }, /Bad var\..*==.*Bad error message/);
+        }, /Bad var\.((.*==)|(.*strictEqual)).*Bad error message/);
```

Comment:
Is it worth it to keep the regular expression simpler, if less strict?:  `/Bad var\..*Bad error message/`?

Reply:
Oops, this was merged. Never mind my question. :-D